frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -274,12 +274,16 @@ export const LINE_SETTINGS = {
   // DEPRECATED: moved to series settings
   "line.marker_enabled": {},
   // DEPRECATED: moved to series settings
-  "line.missing": {
-    default: "interpolate",
-  },
+  "line.missing": {},
```

Comment:
without removing the default value in the legacy setting. We'll always get `interpolated` instead of other values because of this line.

https://github.com/metabase/metabase/blob/9abe75e60175be3178c45bd4c4a5382e9a2dc7dd/frontend/src/metabase/visualizations/shared/settings/series.ts#L75